src/wand/magick.rs

```diff
@@ -875,6 +875,17 @@ impl MagickWand {
         }
     }
 
+    /// Removes skew from the image. Skew is an artifact that
+    /// occurs in scanned images because of the camera being misaligned,
+    /// imperfections in the scanning or surface, or simply because the paper was
+    /// not placed completely flat when scanned
+    pub fn deskew_image(&mut self, threshold: f64) -> Result<()> {
```

Comment:
@nlfiedler — I was unsure of when to make the `self` wand parameter mutable. Seems like for some functions it is marked as mutable and others are not.

When should `self` be marked as mutable for `MagickWand`?